.prettierignore

```diff
@@ -1,6 +1,8 @@
 # Basically, ignore everythings expect app/webpacker/controllers/*.js and app/webpacker/packs/*.js
 *.css
 *.scss
+# Except v2
+!/app/webpacker/css/admin/v2/**/*.scss
```

Comment:
I like the idea of using Prettier more, but once we merge v2 into the main code, we have to remove this line again, and choose:
* a) enable prettier on some or all of the css files, or
* b) not use prettier on css again.

Obviously (a) is the direction to head, but that makes it a pre-requisite for this task.
But it won't hurt to choose (b), at least this new code will be prettier!

Reply:
I think solution `a)` is more satisfying: one solution for hat is probably to create a PR that run prettier on all scss files and then having well formatted files once and for all. 
What do you think?